raft/raftpb/raft.proto

```diff
@@ -15,9 +15,14 @@ enum EntryType {
 }
 
 message Entry {
-	optional EntryType  Type  = 1 [(gogoproto.nullable) = false];
+	// Term MUST use atomic operation to access, keep 64bit aligned.
```

Comment:
Same here,
`optional uint64     Term  = 2 [(gogoproto.nullable) = false]; // must be 64-bit aligned for atomic operations`